czech/marsmelouny.md

```diff
@@ -1,4 +1,4 @@
-Jednoho rána se vzbudíš v divné místnosti se zdmi z maršmelounů.
+Jednoho rána se vzbudíš v divné místnosti se zdmi z maršmelounů. Točí se ti hlava, a tvůj žaludek dává najevo svou nespokojenost. Včera jsi to s těma panákama patrně přehnal. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.